static/bidder-info/adtg_org.yaml

```diff
@@ -0,0 +1,2 @@
+endpoint: "http://ads-pbs.rtb.adtarget.org/openrtb/{{.PublisherID}}?host={{.Host}}"
```

Comment:
@apykhteyev : we should prefer secure http instead of http

Reply:
any reason for s2s integration to use https? and actually all of our aliases were merged with HTTP before

Reply:
@apykhteyev : from docs - https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html
```Bidding server endpoints should prefer secure HTTPS to protect user privacy and should allow keep alive connections (preferably with HTTP/2 support) to increase host performance.```

Reply:
While https is strongly preferred, http is still permitted.